src/Commands/Self/ClearCacheCommand.php

```diff
@@ -5,13 +5,17 @@
 use League\Container\ContainerAwareInterface;
 use League\Container\ContainerAwareTrait;
 use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Helpers\LocalMachineHelper;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
 
 /**
  * Class ClearCacheCommand
  * @package Pantheon\Terminus\Commands\Self
  */
-class ClearCacheCommand extends TerminusCommand implements ContainerAwareInterface
+class ClearCacheCommand extends TerminusCommand implements ConfigAwareInterface, ContainerAwareInterface
 {
+    use ConfigAwareTrait;
```

Comment:
TerminusCommand already uses ConfigAwareTrait. php7 doesn't mind the re-use, but 5.6 and 5.5 don't like it, ergo the test failures.